lms/static/sass/discussion/inline-discussion-rtl.scss

```diff
@@ -1,12 +1,12 @@
 // Discussions - CSS application architecture
 // Version 1 styling (pre-Pattern Library)
 // ====================
+@import 'edx-pattern-library/pattern-library/sass/global/rtl';                    // RTL-specifc settings and utilities
+@import 'edx-pattern-library-loader';
 
-// Load the RTL version of the edX Pattern Library
-$pattern-library-path: '../../edx-pattern-library' !default;
-@import 'edx-pattern-library/pattern-library/sass/edx-pattern-library-rtl';
-
-@import '../build-base-v1-rtl';
+@import '../vendor/bi-app/bi-app-ltr'; // set the layout for right to left languages
+@import '../base/variables-ltr';
```

Comment:
On that note, how do we test RTL languages on apros?  

I tried `?preview-lang=rtl`, which did nothing, and visiting the LMS page `localhost:8000/update_lang/`, which works on upstream, but 404s on solutions.